### ISSUE TITLE
Add RSS 

### ISSUE THREAD

**Issue by eldguru** [remote-wind/remote-wind#166]
Is it possible to add a RSS service?

**Comment by maxcal** [comment]
Yes its not very difficult. Do you want a feed of observations from a single station?

**Comment by eldguru** [comment]
Yes! That's exactly what i'm looking for :-)
